edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/commands/NullCommandFactory.cs

```diff
@@ -23,7 +23,7 @@ public class NullCommandFactory : ICommandFactory
 
         public Task<ICommand> StopAsync(IModule module) => Task.FromResult<ICommand>(NullCommand.Instance);
 
-        public Task<ICommand> RestartAsync(IModule module) => Task.FromResult<ICommand>(NullCommand.Instance);
```

Comment:
Instead of changing the interface, is it possible to create an `IModule` from just it's `id`?

Reply:
Emm, and what would the remaining values be? Null? 
I actually felt requiring an IModule for Restart was not necessary. Same probably goes for Stop as well.

Reply:
I can keep the interface the same and solve it in a diff way...

Reply:
Done